NEWS

```diff
@@ -32,22 +46,16 @@ heatmaply 0.13.0 (2017-11-11)
 
 
 
-
-
-
 heatmaply 0.12.1 (2017-10-25)
 ================
-
-
-### NEW FEATURES
-* heatmaply
-  - Add `cellnote_size` argument, controlling the font size of the cellnote.
-
 ### BUG FIXES
 * heatmaply
   - removing tick labels with `showticklabels` now removes the ticks as
     well as the tick labels.
   - Prevent grid_gap warning (#105)
+  - Add `cellnote_size` argument, controlling the font size of the cellnote.
```

Comment:
I think this is a new feature and not a bug fix (please revert this edit to what was before in the file)